pkg/inttest/http.go

```diff
@@ -109,15 +110,15 @@ func (hc *HTTPClient) Do(t *testing.T, method, path string, requestBody io.Reade
 	return body
 }
 
-// do delegates the request to the underlying HTTP client.
+// do delegate the request to the underlying HTTP client.
```

Comment:
That comment was correct 😅  IMHO as the function `do` is the subject as in `the function do delegates the request`

Same applies to the change in the `Do` methods comment